test/relations.test.js

```diff
@@ -2285,14 +2395,28 @@ describe('relations', function() {
     it('should create polymorphic through model', function(done) {
       PictureLink.findOne(function(err, link) {
         should.not.exist(err);
-        link.pictureId.should.eql(pictures[0].id); // eql for mongo ObjectId
-        link.imageableId.should.eql(author.id);
-        link.imageableType.should.equal('Author');
-        link.imageable(function(err, imageable) {
-          imageable.should.be.instanceof(Author);
-          imageable.id.should.eql(author.id);
-          done();
-        });
+        if (isCassandraConnector) {
+          // Sorting is not set in define for Cassandra
```

Comment:
Can you rewrite this using `containDeep` and share the same asserts for all connectors?